ignite/cmd/plugin_default.go

```diff
@@ -0,0 +1,95 @@
+package ignitecmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	pluginsconfig "github.com/ignite/cli/ignite/config/plugins"
+	"github.com/ignite/cli/ignite/services/plugin"
+)
+
+type defaultPlugin struct {
+	use     string
+	short   string
+	aliases []string
+	path    string
+}
+
+// defaultPlugins holds the plugin that are considered trustable and for which
+// a command will added if the plugin is not already installed.
+// When the user executes that command, the plugin is automatically installed.
+var defaultPlugins = []defaultPlugin{
+	{
+		use:     "network",
+		short:   "Launch a blockchain in production",
+		aliases: []string{"n"},
+		path:    "github.com/ignite/cli-plugin-network@feat/import-from-cli",
```

Comment:
For the moment, version is hard written in this field as is. Note that doesn't prevent the user to upgrade it manually with `plugin update` or by manually update the global config file located in `.ignite/plugins/plugins.yml`

But finding a better version handling is a legitimate question I think.